sysl2/sysl/parse/tests/alias_new.sysl

```diff
@@ -0,0 +1,2 @@
+Model:
```

Comment:
Avoid temporal elements in filenames. It won't be new forever. Suggest `alias_inline.sysl`.